New package: DiplodocusPlots v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: DiplodocusPlots
- Repository: https://github.com/cneverett/DiplodocusPlots.jl
- Created by: @cneverett
- Version: v0.0.1
- Commit: 240b36976c62783552471cce49bc7cb4738c0b3a
- Reviewed by: @cneverett
- Reference: https://github.com/cneverett/DiplodocusPlots.jl/commit/240b36976c62783552471cce49bc7cb4738c0b3a#commitcomment-163719195
- Description: Plotting functionality for the Diplodocus particle transport framework
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd63d9cdca0b93c6d599a3d1dcc1e155b024ec10e616f57c25804e9777f689b347013543f460e4cdc893b85c265dc61e262dff49724fdd6915d5b405295f96570384a253a3e45119e04986e61ad803f49d7f6596a0e4c60005c8a1546069a103a4822e197f71e8b7096999ff85bff77fecdcc1997c988f1f212a22c23beae5ed687594a5ec8246f5d170af87dd27b2e6e954cb9adb61317507045e21a539aa18fff60b9aef47f03a97cfac7e8daf4a2ae99 -->

goerz commented:
Thank you for submitting your package! Could you add a little bit of documentation before registering? [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README. An important part of packages in General is that any potential user can figure out what the package is about and how to get started with using it. That is really difficult when there is no documentation.

In the longer term, I definitely recommend setting up a [Documenter](https://documenter.juliadocs.org/stable/)-based documentation. Before a `v1.0` release, or for smaller packages that can be effectively described entirely with their `README`, that's not a requirement, though.

JuliaRegistrator committed:
New package: DiplodocusPlots v0.0.1

UUID: e2feba3a-3769-4cd4-93ab-3db026c329dd
Repo: https://github.com/cneverett/DiplodocusPlots.jl.git
Tree: 0042869c6255be396771b48d22ddcc45a0697e73

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3

cneverett commented:
I added some basic detail to the README and a link to the inprogress (the online version is outdated compaired to a local version that has the installation instruction and tutorials but is not ready to be released). Still the aim is to update everything with a more public release for v.0.0.2

cneverett commented:
[no block]
